packages/react/src/components/Forms/Switch/index.stories.tsx

```diff
@@ -0,0 +1,191 @@
+import { Meta } from "@storybook/react";
+import React from "react";
+import { Switch } from ":/components/Forms/Switch/index";
+import { Button } from ":/components/Button";
+
+export default {
+  title: "Components/Forms/Switch",
+  component: Switch,
+} as Meta<typeof Switch>;
+
+export const Default = {
+  args: {},
+};
+
+export const Checked = {
+  args: {
+    checked: true,
+  },
+};
+
+export const WithLabel = {
+  args: {
+    label: "Label",
+  },
+};
+
+export const WithLabelChecked = {
+  args: {
+    label: "Label",
+    checked: true,
+  },
+};
+
+export const WithText = {
+  args: {
+    label: "Label",
+    text: "This is an optional text",
+    checked: true,
+  },
+};
+
+export const FullWidth = {
+  args: {
+    label: "Label",
+    text: "This is an optional text",
+    fullWidth: true,
+  },
+};
+
+export const WithLabelRight = {
+  args: {
+    label: "Label",
+    labelSide: "right",
+  },
+};
+
+export const WithLabelRightAndText = {
+  args: {
+    label: "Label",
+    labelSide: "right",
+    text: "This is an optional text",
+  },
+};
+
+export const WithLabelRightAndFullWidth = {
+  args: {
+    label: "Label",
+    text: "This is an optional text",
+    fullWidth: true,
+    labelSide: "right",
+  },
+};
+
+export const Disabled = {
+  args: {
+    label: "Label",
+    text: "This is an optional text",
+    disabled: true,
+  },
+};
+
+export const DisabledChecked = {
+  args: {
+    label: "Label",
+    text: "This is an optional text",
+    disabled: true,
+    defaultChecked: true,
+  },
+};
+
+export const Error = {
+  args: {
+    label: "Label",
+    text: "This is an optional text",
+    state: "error",
+    defaultChecked: true,
+  },
+};
+
+export const Success = {
+  args: {
+    label: "Label",
+    text: "This is an optional text",
+    state: "success",
+    defaultChecked: true,
+  },
+};
```

Comment:
By checking this story, I realized that field success text contrast is really too low... 2.52:1 instead of 4.5:1.

To get a correct contrast, we should use `colors--success-800`, I test and it could be ok, WDYT ?

<img width="140" alt="image" src="https://github.com/openfun/cunningham/assets/9265241/4cbe3da6-1a2d-44a5-9961-3d0d421c9dd6">

Reply:
Here is `success-800` 

<img width="113" alt="Capture d’écran 2023-05-24 à 14 14 22" src="https://github.com/openfun/cunningham/assets/9628870/e6ec9218-4278-419d-be3e-f22956915347">

I find it really dark and not that joyful. 

But we could do what we did with checkboxes and radio, use `success-700` like this:

<img width="148" alt="Capture d’écran 2023-05-24 à 14 15 19" src="https://github.com/openfun/cunningham/assets/9628870/765956ac-9d03-471b-8faf-0369d3398e01">

WDYT?

Reply:
Oh my point was to only change the success text label !

Reply:
I think it should be the same color because it both match the "it's correct" signal